src/site/content/en/blog/form-fatigue-validate-user-input/index.md

```diff
@@ -0,0 +1,308 @@
+---
+title: "Reduce form fatigue: validate user input"
+author: mdiblasio
+date: 2019-05-02
+description: |
+  This post describes how to validate user input.
+wf_blink_components: Blink>Accessibility
+tags:
+  - post
+  - accessibility
+  - ux
+---
+
+_This is post 4 of 5 of the [Reduce form fatigue series](../form-fatigue) that
+shows you how to build better forms for online stores._
+
+<!-- TODO (robdodson): is there a format we should use for this line that opens each article?  -->
+
+<!-- TODO (dutton): is there a format we should use for this line that opens each article?  -->
+
+No matter how optimized your forms are, users will make mistakes! When this
+happens, provide them with helpful feedback in real-time so they can quickly fix
+the issue. 
+
+## Restrict user input
+
+An effective way to avoid validation errors is to restrict user input (where
+possible) using built-in attributes.
+
++   **For numeric input**, use the `min` and `max` attributes to limit
```

Comment:
Suggest removing the bullets throughout. I think the text reads nicely without them.